File: ClinicSystem/src/com/example/clinicsystem/models/classes/M_Schedule.java
package com.example.clinicsystem.models.classes;

import com.example.clinicsystem.helpers.DatabaseConnection;

import java.sql.ResultSet;
import java.sql.SQLException;
import java.util.ArrayList;
import java.util.List;

public class M_Schedule {
    protected int ID;
    protected String date;

    public M_Schedule(int ID, String date) {
        this.ID = ID;
        this.date = date;
    }

    public M_Schedule() {
    }

    public List<M_Schedule> getAllSchedules(String condition) {
        List<M_Schedule> schedules = new ArrayList<>();
        DatabaseConnection databaseConnection = DatabaseConnection.getINSTANCE();
        ResultSet resultSet = databaseConnection.select("schedule", condition);

        if(resultSet != null) {
            try {
                while (resultSet.next()) {
                    int id = resultSet.getInt("Schedule_ID");
                    String date = resultSet.getString("Schedule_Date");

                    M_Schedule schedule = new M_Schedule(id, date);

                    schedules.add(schedule);
                }
            } catch (SQLException e) {
                e.printStackTrace();
            }
        }

        return schedules;
    }

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
}

File: clinic/M_Schedule.php
<?php
include_once "DatabaseConnection.php";

class M_Schedule {
	protected $id;
    protected $sDate;
	
	public function __construct($id, $date) {
		$this->id=$id;
		$this->sDate=$date;
	}
	
	public function addSchedule() {
        $conn = DatabaseConnection::getINSTANCE();

        return $conn->insertInto("schedule", "(Schedule_Date)",
			"('" + this.date + "')");
    }

    public function getAllSchedules($condition) {
		$conn = DatabaseConnection::getINSTANCE();
		
		$result = $conn->select("schedule", $condition);
		$schedules = array();
		
		while ($row = $result->fetch_assoc()) {
			$schedule = new M_Schedule($row["Schedule_ID"], $row["Schedule_Date"]);
			array_push($schedules, $schedule);
		}
		
		return $schedules;
    }

    public function getID() {
        return $this->id;
    }

    public function setID($id) {
        $this->id = $id;
    }

    public function getsDate() {
        return $this->sDate;
    }

    public function setsDate($date) {
        $this->SsDate = $date;
    }
}

?>